erp/mapper/vaccination.py

```diff
@@ -0,0 +1,211 @@
+from datetime import datetime
+
+from django.contrib.gis.geos import Point
+
+from core.lib import text
+from erp.models import Accessibilite, Commune, Erp
+from erp.provider import arrondissements
+
+
+class RecordMapper:
+    FIELDS_MAP = {
+        "c_nom": "nom",
+        "c_adr_num": "numero",
+        "c_adr_voie": "voie",
+        "c_com_nom": "commune",
+        "c_com_cp": "code_postal",
+        "c_com_insee": "code_insee",
+        "c_rdv_tel": "telephone",
+    }
+
+    RESERVE_PS = [
```

Comment:
"PS" is not obvious, maybe change it to "PROS" or "PROFESSIONALS"

Reply:
Addressed in https://github.com/MTES-MCT/acceslibre/pull/136/commits/71bc108f68b7438bafd66b7e206b0167e3ee0452